src/components/MiniMap.vue

```diff
@@ -1,22 +1,39 @@
 <template>
-  <div class="mini-paper-container" @click="movePaper" :class="isOpen ? 'opened' : 'closed'">
-    <div ref="miniPaper" class="mini-paper" />
-  </div>
+  <b-button
+    class="btn btn-sm btn-secondary mini-map-btn ml-auto"
+    data-test="mini-map-btn"
+    @click="isOpen = !isOpen"
+    v-b-tooltip.hover
+    :title="isOpen ? $t('Hide Mini-Map') : $t('Show Mini-Map')"
+  >
+    <font-awesome-icon :icon="miniMapOpen ? minusIcon : mapIcon" />
+    <div class="mini-paper-container" @click="movePaper" :class="{ 'opened' : isOpen,  'closed': !isOpen, 'expanded' : expanded }">
+      <div ref="miniPaper" class="mini-paper" />
+    </div>
+  </b-button>
```

Comment:
These changes combine the mini-map and the buttons/icons that control it. This works as the mini-map is absolutely positioned.